.github/workflows/ci.yml

```diff
@@ -0,0 +1,83 @@
+name: Continuous Integration
```

Comment:
I'm on the fence on that one as I personally push commits frequently and want CI to run through each one in case something in a specific commits breaks build. In those cases it is much easier narrowing down what change broke build. The counter argument is that the less time build takes the less likely it is that a situation like that would occur. I want to see what others think cc @dekz @dextracker @kyu-c

Reply:
I don't have a strong opinion 
Most of the time, I run test locally and only use the CI result as the final sanity check.